src/app/form/(formSteps)/personal-details/2/PersonalDetailsStep2.test.tsx

```diff
@@ -69,6 +67,11 @@ describe("<PersonalDetailsStep2 />", () => {
       await user.selectOptions(combobox, "PA");
       expect(combobox).toHaveValue("PA");
     });
+
+    it("enables autocompleting address", () => {
```

Comment:
dust - This sounds a little off to my ear, is this more correct?
```suggestion
    it("enables autocompleting an address", () => {
```
or something else?